src/components/Layout/Layout.tsx

```diff
@@ -5,7 +5,7 @@ import { Button, ButtonGroup } from '@mui/material';
 import useWindowSize from '@/utils/useWindowSize';
 import HamburgerDropDown from '../HambergerDropdown/HambergerDropDown';
 import RenderIf from '../RenderIf/RenderIf';
-import { colors, mediaQueriesThresholds, headerNavigation } from '../../constants/constants';
+import { colors, mediaQueriesThresholds, headerNavigation, font } from '../../constants/constants';
```

Comment:
As with my previous comment, not everything is a constant. Doing constants/constants is also super redundant. Take advantage of index.ts at the least.